data-prepper-script/build.gradle

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
In line with my comment below, it may be good to rename this to `data-prepper-expression` to limit scope to expressions.